src/nxdoc/nuxeo-server/installation/nuxeo-cluster-architecture/nuxeo-architecture-components/database-architecture.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Database
+description:
+labels:
+    - deployment
+    - bchauvin
+    - architecture
+    - cluster
+    - content-review-lts2017
+    - lts2019-ok
+review:
+    comment: ''
+    date: '2021-03-10'
+    status: ok
+toc: true
+tree_item_index: 500
+---
+
+## Concept
+
+The database is a core component for your Nuxeo cluster infrastructure, since it will store the document hierarchy, all document properties, and will be used as well for various queries. Nuxeo supports many databases, both NoSQL and relational ones. Among them, MongoDB and PostgreSQL currently provide the best overall performances for Nuxeo usage.
+
+## Recommendation
+
+A database system providing high availability. Each solution has its own options for this, therefore we can't go into further details here.
+
+Best performances are obtained using **MongoDB** because it is a document based database. It is built to manage large volumes and is highly scalable. Moreover, using MongoDB allows you to manage key/value stores, and therefore avoid a Redis deployment.
```

Comment:
```suggestion
Best performances are obtained using **MongoDB** because it is a document based database. It is built to manage large volumes and is highly scalable.
```
Even a SQL DB can store the KV.